dialogue-clients/src/main/java/com/palantir/dialogue/clients/DialogueClients.java

```diff
@@ -71,7 +72,10 @@ public static ReloadingFactory create(Refreshable<ServicesConfigBlock> scb) {
                     WithDialogueOptions<ReloadingFactory>,
                     ConjureClients.NonReloadingClientFactory,
                     ConjureClients.ToReloadingFactory<ReloadingFactory>,
-                    ReloadingChannelFactory {}
+                    ReloadingChannelFactory {
+        // TODO(dfox): c-j-r should really define this method in WithClientOptions
```

Comment:
I think I just wanna get this right from the beginning. https://github.com/palantir/conjure-java-runtime/pull/1592